docs/cli/templates.md

```diff
@@ -267,7 +267,9 @@ module.exports = (event, context) => {
 ```
 
 ##### Node.js 12 `node12` - Access to the raw body
-Set the environment variable `RAW_BODY` to `true` to set the `context.body` to the request body buffer rather than parsing it as JSON
+Set the environment variable `RAW_BODY` to `true` to set the `context.body` to the request body buffer rather than parsing it as JSON.
```

Comment:
I think "to the original request body" might make this just a little bit clearer.

It might also be good to indicate the JSON parsing is the default "parsing it as JSON, the default behavior." 

I think these changes will help any reader that is quickly scrolling or skimming

Reply:
Good points. Thank you. I'll update that this evening